plugins/arDominionB5Plugin/scss/_layout.scss

```diff
@@ -123,3 +123,93 @@ body > iframe {
 #front-map {
   height: 200px;
 }
+
+#treeview {
+  max-height: 36rem;
+  overflow: hidden auto;
+
+  // Sortable placeholder.
+  .placeholder {
+    background-color: rgba(255, 255, 255, 0.8);
```

Comment:
This looks like it can do the trick:
```
@use "sass:color";
background-color: color.adjust($white, $alpha: -0.2);
```

Reply:
I was thinking on Bootstrap variables instead of the Sass approach. Looking at their docs they use `rgba` so `rgba($white, 0.8)` may be cleaner in the end, without the need of `@use`.

https://getbootstrap.com/docs/5.0/utilities/colors/#utilities-api

Sorry for the back and forward, and not a big deal of course, feel free to squash and merge ;)

Reply:
Cool, that worked. Fixed.